uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +70,22 @@ protected function cleanup() {
     }
   }
 
+  private function loadRequiredData() {
```

Comment:
It actually depends on where this data comes from. It's safe if it comes from either:
- Any of these files https://github.com/civicrm/civicrm-core/blob/728bbd5bbad0b5269dcd0a4cb70384291d309e70/Civi/Test/Data.php#L20-L22 (These are loaded and the db schema is reset for the tests)
- Added by the extension (as you said)